src/components/CookieNotice/CookiePolicy.scss

```diff
@@ -0,0 +1,48 @@
+@import "constants/style.scss";
+
+$cookie-consent-button-text-size: 18px;
+
+.portal {
+  border-radius: 5px;
+}
+
+.cookie-policy {
+  margin: $margin--small;
+}
+
+.cookie-policy__title {
+  color: $color-black;
+  background-color: $color-light-gray;
+  padding: 5px $padding--default 5px $padding--default;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+}
+
+.cookie-policy__body {
+  color: $color-black;
+  background-color: $color-light-gray;
+  padding: $padding--default;
+  height: 375px;
+  overflow: auto;
+}
+
+.cookie-policy__footer {
+  background-color: $color-light-gray;
+  padding: $padding--default;
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+.cookie-policy__button-accept {
+  background: $color-backlog-blue;
+  color: $color-white;
+  margin: 0 0 0 auto;
+  border: none;
+  padding: 5px 15px;
+  border-radius: 5px;
+  font-family: inherit;
+  font-size: $cookie-consent-button-text-size;
+  justify-content: flex-end;
+  display: flex;
```

Comment:
`justify-content: flex-end` doesn't do anything here because you do not position any element inside of the button. Also, just for placing the button at the end of the footer I'd use `display: block` or `display: inline-block`